src/Domain.Linnapps/SernosConfig.cs

```diff
@@ -1,9 +1,12 @@
 ﻿namespace Linn.Products.Domain.Linnapps
 {
+    using System.ComponentModel.DataAnnotations;
+    using System.ComponentModel.DataAnnotations.Schema;
     using System.Linq;
 
     using Linn.Products.Domain.Linnapps.Exceptions;
 
+    [Table("SERNOS_CONFIG")]
```

Comment:
not sure about using the data annotation to do the database mapping as it brings persistence into the domain layer when you can do it via the DbContext class in a fluent way